// The expected format of this file can be found at https://github.com/dotnet/coreclr/blob/master/Documentation/building/testing-with-corefx.md
[
    {
        "name": "System.Collections.Tests",
        "exclusions": {
            "namespaces": null,
            "classes": null,
            "methods": [
                {
                    "name": "System.Collections.Tests.SortedList_Generic_Tests_string_string.ICollection_Generic_Remove_DefaultValueContainedInCollection",
                    "reason": "Exclusion Sample"
                }
            ]
        }
    },
    {
        "name": "System.Runtime.Extensions.Tests",
        "exclusions": {
            "namespaces": null,
            "classes": null,
            "methods": []
        }
    },
    {
        "name": "System.Runtime.Tests",
        "exclusions": {
            "namespaces": null,
            "classes": null,
            "methods": []
        }
    },
    {
        "name": "System.Threading.Tasks.Tests",
        "exclusions": {
            "namespaces": null,
            "classes": null,
            "methods": []
        }
    },
    {
        "name": "System.Threading.Tests",
        "exclusions": {
            "namespaces": null,
            "classes": null,
            "methods": []
        }
    },
    {
        "name": "System.Collections.Tests",
        "enabled": true,
        "exclusions": {
            "namespaces": null,
            "classes": null,
            "methods": [
                {
                    "name": "System.Collections.Tests.Dictionary_Generic_Tests_string_string.IEnumerable_Generic_Enumerator_Reset_ModifiedBeforeEnumeration_ThrowsInvalidOperationException",
                    "reason": "Assert.All() Failure: 1 out of 4 items in the collection did not pass.\r\n"
                },
                {
                    "name": "System.Collections.Tests.Dictionary_Generic_Tests_string_string.IEnumerable_Generic_Enumerator_Reset_ModifiedAfterEnumeration_ThrowsInvalidOperationException",
                    "reason": "Assert.All() Failure: 1 out of 4 items in the collection did not pass.\r\n"
                },

                {
                    "name": "System.Collections.Tests.Dictionary_Generic_Tests_string_string.IEnumerable_Generic_Enumerator_MoveNext_ModifiedAfterEnumeration_ThrowsInvalidOperationException",
                    "reason": "Assert.All() Failure: 1 out of 4 items in the collection did not pass.\r\n"
                },
                {
                    "name": "System.Collections.Tests.Dictionary_Generic_Tests_string_string.IEnumerable_Generic_Enumerator_MoveNext_ModifiedBeforeEnumeration_ThrowsInvalidOperationException",
                    "reason": "Assert.All() Failure: 1 out of 4 items in the collection did not pass.\r\n"
                },
                {
                    "name": "System.Collections.Tests.Dictionary_Generic_Tests_string_string.IEnumerable_Generic_Enumerator_Reset_ModifiedDuringEnumeration_ThrowsInvalidOperationException",
                    "reason": "Assert.All() Failure: 1 out of 4 items in the collection did not pass.\r\n"
                },
                {
                    "name": "System.Collections.Tests.Dictionary_Generic_Tests_string_string.IEnumerable_Generic_Enumerator_MoveNext_ModifiedDuringEnumeration_ThrowsInvalidOperationException",
                    "reason": "Assert.All() Failure: 1 out of 4 items in the collection did not pass.\r\n"
                },
                {
                    "name": "System.Collections.Tests.Dictionary_Generic_Tests_SimpleInt_int_With_Comparer_WrapStructural_SimpleInt.IEnumerable_Generic_Enumerator_Reset_ModifiedBeforeEnumeration_ThrowsInvalidOperationException",
                    "reason": "Assert.All() Failure: 1 out of 4 items in the collection did not pass.\r\n"
                },
                {
                    "name": "System.Collections.Tests.Dictionary_Generic_Tests_SimpleInt_int_With_Comparer_WrapStructural_SimpleInt.IEnumerable_Generic_Enumerator_MoveNext_ModifiedDuringEnumeration_ThrowsInvalidOperationException",
                    "reason": "Assert.All() Failure: 1 out of 4 items in the collection did not pass.\r\n"
                },
                {
                    "name": "System.Collections.Tests.Dictionary_Generic_Tests_SimpleInt_int_With_Comparer_WrapStructural_SimpleInt.IEnumerable_Generic_Enumerator_MoveNext_ModifiedBeforeEnumeration_ThrowsInvalidOperationException",
                    "reason": "Assert.All() Failure: 1 out of 4 items in the collection did not pass.\r\n"
                },
                {
                    "name": "System.Collections.Tests.Dictionary_Generic_Tests_SimpleInt_int_With_Comparer_WrapStructural_SimpleInt.IEnumerable_Generic_Enumerator_Reset_ModifiedDuringEnumeration_ThrowsInvalidOperationException",
                    "reason": "Assert.All() Failure: 1 out of 4 items in the collection did not pass.\r\n"
                },
                {
                    "name": "System.Collections.Tests.Dictionary_Generic_Tests_SimpleInt_int_With_Comparer_WrapStructural_SimpleInt.IEnumerable_Generic_Enumerator_Reset_ModifiedAfterEnumeration_ThrowsInvalidOperationException",
                    "reason": "Assert.All() Failure: 1 out of 4 items in the collection did not pass.\r\n"
                },
                {
                    "name": "System.Collections.Tests.Dictionary_Generic_Tests_SimpleInt_int_With_Comparer_WrapStructural_SimpleInt.IEnumerable_Generic_Enumerator_MoveNext_ModifiedAfterEnumeration_ThrowsInvalidOperationException",
                    "reason": "Assert.All() Failure: 1 out of 4 items in the collection did not pass.\r\n"
                },
                {
                    "name": "System.Collections.Tests.Dictionary_Generic_Tests_int_int.IEnumerable_Generic_Enumerator_Reset_ModifiedDuringEnumeration_ThrowsInvalidOperationException",
                    "reason": "Assert.All() Failure: 1 out of 4 items in the collection did not pass.\r\n"
                },
                {
                    "name": "System.Collections.Tests.Dictionary_Generic_Tests_int_int.IEnumerable_Generic_Enumerator_MoveNext_ModifiedAfterEnumeration_ThrowsInvalidOperationException",
                    "reason": "Assert.All() Failure: 1 out of 4 items in the collection did not pass.\r\n"
                },
                {
                    "name": "System.Collections.Tests.Dictionary_Generic_Tests_int_int.IEnumerable_Generic_Enumerator_MoveNext_ModifiedBeforeEnumeration_ThrowsInvalidOperationException",
                    "reason": "Assert.All() Failure: 1 out of 4 items in the collection did not pass.\r\n"
                },
                {
                    "name": "System.Collections.Tests.Dictionary_Generic_Tests_int_int.IEnumerable_Generic_Enumerator_Reset_ModifiedBeforeEnumeration_ThrowsInvalidOperationException",
                    "reason": "Assert.All() Failure: 1 out of 4 items in the collection did not pass.\r\n"
                },
                {
                    "name": "System.Collections.Tests.Dictionary_Generic_Tests_int_int.IEnumerable_Generic_Enumerator_Reset_ModifiedAfterEnumeration_ThrowsInvalidOperationException",
                    "reason": "Assert.All() Failure: 1 out of 4 items in the collection did not pass.\r\n"
                },
                {
                    "name": "System.Collections.Tests.Dictionary_Generic_Tests_int_int.IEnumerable_Generic_Enumerator_MoveNext_ModifiedDuringEnumeration_ThrowsInvalidOperationException",
                    "reason": "Assert.All() Failure: 1 out of 4 items in the collection did not pass.\r\n"
                },
                {
                    "name": "System.Collections.Tests.Dictionary_IDictionary_NonGeneric_Tests.IEnumerable_NonGeneric_Enumerator_MoveNext_ModifiedBeforeEnumeration_ThrowsInvalidOperationException",
                    "reason": "Assert.All() Failure: 1 out of 4 items in the collection did not pass.\r\n"
                },
                {
                    "name": "System.Collections.Tests.Dictionary_IDictionary_NonGeneric_Tests.IEnumerable_NonGeneric_Enumerator_Reset_ModifiedBeforeEnumeration_ThrowsInvalidOperationException",
                    "reason": "Assert.All() Failure: 1 out of 4 items in the collection did not pass.\r\n"
                },
                {
                    "name": "System.Collections.Tests.Dictionary_IDictionary_NonGeneric_Tests.IEnumerable_NonGeneric_Enumerator_Reset_ModifiedDuringEnumeration_ThrowsInvalidOperationException",
                    "reason": "Assert.All() Failure: 1 out of 4 items in the collection did not pass.\r\n"
                },
                {
                    "name": "System.Collections.Tests.Dictionary_IDictionary_NonGeneric_Tests.IEnumerable_NonGeneric_Enumerator_MoveNext_ModifiedDuringEnumeration_ThrowsInvalidOperationException",
                    "reason": "Assert.All() Failure: 1 out of 4 items in the collection did not pass.\r\n"
                },
                {
                    "name": "System.Collections.Tests.Dictionary_IDictionary_NonGeneric_Tests.IEnumerable_NonGeneric_Enumerator_Reset_ModifiedAfterEnumeration_ThrowsInvalidOperationException",
                    "reason": "Assert.All() Failure: 1 out of 4 items in the collection did not pass.\r\n"
                },
                {
                    "name": "System.Collections.Tests.Dictionary_IDictionary_NonGeneric_Tests.IEnumerable_NonGeneric_Enumerator_MoveNext_ModifiedAfterEnumeration_ThrowsInvalidOperationException",
                    "reason": "Assert.All() Failure: 1 out of 4 items in the collection did not pass.\r\n"
                }
            ]
        }
    }
]
